package.json

```diff
@@ -25,7 +25,7 @@
   "peerDependencies": {
     "eslint": "^8.34.0",
     "prettier": "^1.19.0 || ^2.0.0",
-    "typescript": "^3.7.0 || ^4.0.0"
+    "typescript": "^3.7.0 || ^4.0.0 || ^5.0.0"
   },
   "dependencies": {
     "@typescript-eslint/eslint-plugin": "^5.59.9",
```

Comment:
Do we need to upgrade our `typescript-eslint` parser or anything to support the newest syntax?

Reply:
That's a good point. i'm not actually using any TS 5 features when I'm testing.

skimming the change log they do introduce higher version of TS support in more recent versions of these libs. So I'll bump these and test again 👍

Reply:
I updated most of the packages in here. the one you mentioned did need to be updated for more recent versions of TS 5. so good flag!

I tested this in SOS with v4.9.5 and with v5.2.2 (though i literally just bumped the TS lib and ran lint again and made sure there were no errors or version conflicts)